static/templates/unstar-messages-modal.handlebars

```diff
@@ -0,0 +1,13 @@
+<div id="unstar-messages-modal" class="modal new-style modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="unstar_messages_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="unstar_messages_modal_label">{{t "Unstar all messages" }}</h3>
+    </div>
+    <div class="modal-body">
+        <p>{{t "This action is permanent and cannot be undone. All starred messages will get unstarred. Are you sure, you want to continue this action?" }}</p>
```

Comment:
@rishig can you do a pass on this text?  I think that's the main/only thing left with this PR.